packages/eas-cli/src/credentials/manager/ManageAndroid.ts

```diff
@@ -163,29 +161,21 @@ export class ManageAndroid {
     );
     const appLookupParams = await getAppLookupParamsFromContextAsync(ctx, gradleContext);
     if (action === AndroidActionType.CreateKeystore) {
-      const selectBuildCredentialsResult = await new SelectAndroidBuildCredentials(
+      const keystore = await new CreateKeystore(appLookupParams.account).runAsync(ctx);
+      const selectBuildCredentialsResult = await new CreateAndroidBuildCredentials().runAsync();
```

Comment:
I think that currently we first:
1. create keystore
2. select build credentials
but previously we:
1. selected build credentials
2. created the keystore

Is this change of order of operations necessary?